importers/test/test_djornl_parser.py

```diff
@@ -35,6 +35,18 @@ def init_parser_with_path(self, root_path):
             parser._configure()
             return parser
 
+    def test_errors(self, parser=None, errs={}):
+        if parser is None:
+            self.assertTrue(True)
+            return
+
+        for data_type in errs.keys():
+            with self.subTest(data_type=data_type):
```

Comment:
data_type here is one of `edges`, `nodes` or `clusters`, so `method` is `load_edges`, `load_nodes` or `load_clusters`